.github/workflows/ci.yaml

```diff
@@ -278,7 +282,8 @@ jobs:
       (github.event_name == 'push') &&
       (github.ref == 'refs/heads/main' ||
        startsWith(github.ref, 'refs/heads/release-') ||
-       startsWith(github.ref, 'refs/tags/v'))
+       startsWith(github.ref, 'refs/tags/v')) &&
+      (github.event_name == 'workflow_dispatch')
```

Comment:
If I read this correctly, the push images step will run only when the event name is 'workflow-dispatch' - which changes the current behavior and looks wrong. Maybe you mean to skip push for nightly build.

But since we already have this rule:

    github.event_name == 'push'

I don't think we need to add anything, since nightly build will never have this event name.

I think when the issue was opened we did not have the same rules.